cmd/gb-vendor/vendor/repo.go

```diff
@@ -163,7 +163,8 @@ func probeGitUrl(schemes []string, host, path string, insecure bool) (string, er
 		switch scheme {
 		case "https":
 			url := scheme + "://" + host + "/" + path
-			if _, err := run("git", "ls-remote", "--exit-code", url, "HEAD"); err == nil {
```

Comment:
i had at look at what the go tool does and it effectively does this, but does not include "--exit-code". What about just dropping the --exit-code flag ? Will that suffice ?